### ISSUE TITLE
Triangulation_2 includes a file from Triangulation_3

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#1483]
The three following files:
```
Triangulation_2/include/CGAL/Delaunay_triangulation_2.h
Triangulation_2/include/CGAL/Constrained_Delaunay_triangulation_2.h
Triangulation_2/include/CGAL/Regular_triangulation_2.h
```
include `Triangulation_3/include/CGAL/internal/info_check.h`.

**Comment by afabri** [comment]
I suggest to duplicate the file as it contains  10 lines, and as it is related to   `..with_info()`  which we only have in the triangulation classes.

**Comment by lrineau** [comment]
Let's move the file to `STL_Extension`, like `STL_Extension/include/CGAL/Has_timestamp.h`. It's almost the same usage.

**Comment by lrineau** [comment]
Or let's move it to Triangulation_2, as Triangulation_3 already depends on T2.

**Comment by afabri** [comment]
And let's rename it to Has_info.h

**Comment by lrineau** [comment]
Get_info.h actually.

**Comment by afabri** [comment]
As fist step moved it from T3 to T2, without any renaming, in the branch lrineau/Installation-compute_dependencies-GF

**Comment by lrineau** [comment]
Fixed by 91c6426 from PR #1530.
